src/views/Events.svelte

```diff
@@ -3,9 +3,11 @@ import EventFilters from '../components/EventFilters.svelte'
 import FilteredDisplay from '../components/FilteredDisplay.svelte'
 import Icon from 'fa-svelte'
 import { faExternalLinkAlt } from '@fortawesome/free-solid-svg-icons'
-import { populateEventFilterFrom } from '../data/eventFiltering'
-import { events, loading } from '../data/events'
+import { isParticipant, populateEventFilterFrom } from '../data/eventFiltering'
+import { addToMyEvents, events, loading } from '../data/events'
```

Comment:
based my earlier feedback about exporting `join`, if you're tied to `addToMyEvents` here, which I could see, you could simply alias it, e.g., `join as addToMyEvents`